### PR TITLE
Add fund wallet functionality

### PR DESCRIPTION
**Pull request by josh-richardson**
Adds the ability to request funds on the Rinkeby testnet from the Web3j faucet.

### CODE REVIEW

File: src/main/java/org/web3j/console/WalletFunder.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                            i -> {
+                                if (found.get()) return;
+                                String potentialHash =
+                                        Numeric.toHexString(
+                                                        sha256(
+                                                                (i + config.seed)
+                                                                        .getBytes(
+                                                                                StandardCharsets
+                                                                                        .UTF_8)))
+                                                .substring(2);
```
Comment (josh-richardson) [inline src/main/java/org/web3j/console/WalletFunder.java]: Is this the standard way spotless formats this or is it a bug?

Comment (cfelde) [reply src/main/java/org/web3j/console/WalletFunder.java]: I was about to ask..

**Comment by cfelde** [comment]
How obvious is it that this will fund an address on Rinkeby? Do we need to highlight this? What about other testnets?

**Comment by josh-richardson** [comment]
> How obvious is it that this will fund an address on Rinkeby? Do we need to highlight this? What about other testnets?

I've added a line which will make obvious that this fund operation is on Rinkeby. The faucet server-side doesn't currently support any other testnets, but this is something we could look into in future.

File: src/main/java/org/web3j/console/WalletFunder.java [hunk 2]
```diff
@@ -36,17 +36,24 @@
...
+    public static void main(IODevice console, String[] args) {
         if (args.length != 1 && args.length != 3) {
             exitError(USAGE);
         }
 
         try {
+            String fund =
+                    console.readLine(
+                            "This command will fund the specified wallet on the Rinkeby testnet. Do you wish to continue? [Y/n]: ");
+            if (!fund.toUpperCase().equals("Y")) {
```
Comment (cfelde) [inline src/main/java/org/web3j/console/WalletFunder.java]: I believe traditionally an upper case Y in a Y/n prompt indicates Y being the default option if nothing is entered.

So this should progress if the user types lower or upper case Y, but also progress when nothing is entered (blank).

**Pull request merged**